openslides/motions/static/templates/motions/motion-detail/view-original.html

```diff
@@ -46,3 +46,8 @@
         </div>
     </li>
 </ul>
+
+<div class="tooltip right tt_change_recommendation_create_help" role="tooltip">
```

Comment:
please use `uib-tooltip` from angular-bootstrap (see https://angular-ui.github.io/bootstrap/#/tooltip or pull request #2680).

The tooltip position to left and right are really not very usefule...Suggestion: Show tooltop to top so that the tooltop does not overlap the following text lines.